addons/account/tests/common.py

```diff
@@ -682,6 +682,13 @@ def assertNodeDictEqual(node_dict, expected_node_dict):
             self._turn_node_as_dict_hierarchy(expected_xml_tree),
         )
 
+    def assertMoveName(self, move, expected_name):
```

Comment:
Not sure if we need this. But if we need it, shouldn't it be only in `addons/account/tests/test_sequence_mixin.py`?